### PR TITLE
support samples_types_units column

### PR DESCRIPTION
**Pull request by akvlad**
new `samples_types_units` column

### CODE REVIEW

File: exporter/clickhouseprofileexporter/ch/access_native_columnar.go [hunk 1]
```diff
@@ -53,6 +72,7 @@ func (ch *clickhouseAccessNativeColumnar) InsertBatch(ls plog.Logs) error {
 	timestamp_ns := make([]uint64, sz)
 	typ := make([]string, sz)
 	service_name := make([]string, sz)
+	sample_types_units := make([][]any, sz)
```
Comment (wtomershafir) [inline exporter/clickhouseprofileexporter/ch/access_native_columnar.go:75]: maybe a special type and not any?

File: receiver/pyroscopereceiver/receiver.go [hunk 2]
```diff
@@ -329,8 +329,26 @@ func resetHeaders(req *http.Request) {
...
+	}
+	return res
+}
+
 func setAttrsFromProfile(prof profile_types.ProfileIR, m pcommon.Map) {
 	m.PutStr("type", prof.Type.Type)
+	s := m.PutEmptySlice("sample_types")
+	err := s.FromRaw(stringToAnyArray(prof.Type.SampleType))
+	if err != nil {
+		panic(err)
```
Comment (wtomershafir) [inline receiver/pyroscopereceiver/receiver.go:345]: should panic here?

File: exporter/clickhouseprofileexporter/ch/access_native_columnar.go (hunk 1 above)
Comment (akvlad) [reply exporter/clickhouseprofileexporter/ch/access_native_columnar.go:75]: here should be any due to clickhouse needs.

File: receiver/pyroscopereceiver/receiver.go (hunk 2 above)
Comment (akvlad) [reply receiver/pyroscopereceiver/receiver.go:345]: here should not panic. Will fix.

File: exporter/clickhouseprofileexporter/ch/access_native_columnar.go (hunk 1 above)
Comment (tomershafir) [reply exporter/clickhouseprofileexporter/ch/access_native_columnar.go:75]: cant it be like `tags := make([][]tuple, sz)`?

Comment (akvlad) [reply exporter/clickhouseprofileexporter/ch/access_native_columnar.go:75]: tuple have always been resolving to []any in clickhouse golang drivers.
If you can provide examples of another logic, feel free to put an example.

Comment (akvlad) [reply exporter/clickhouseprofileexporter/ch/access_native_columnar.go:75]: I see. You want me to typedef []any to tuple. In spite of it's an unneeded syntactic sugar, I'll try. :)

Comment (tomershafir) [reply exporter/clickhouseprofileexporter/ch/access_native_columnar.go:75]: either add the sugar or remove it entirely :)

**Pull request merged**